api/handlers.go

```diff
@@ -341,6 +341,29 @@ func getVolumes(c *context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	danglingOnly := false
+	refVolumes := make(map[string]bool)
+
+	if filters.Include("dangling") {
```

Comment:
Actually I do not like so much ident. Could rewrite this to use return fast to make it much more readable?

if